.github/ISSUE_TEMPLATE/bug_report_form.yml

```diff
@@ -0,0 +1,77 @@
+name: 🐛 Bug Report
+description: File a bug report
+title: "🐛 |  "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        
+        🙌 Please, make sure your bug report is unique 🦄 ! 
+        Did you search issues labeled with Bug ? If you find anything relevant to you, up-vote 👍 or add a comment 💬 !
+        
+        For more information, please visit our [contribute section](https://github.com/chutney-testing/chutney/blob/master/CONTRIBUTING.md)
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Describe the bug - What is the current behavior?
+      placeholder: A clear and concise description of what actually happens
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: How one can reproduce the issue ?
+      placeholder: This is very important - Use GIFs to record behavior if applicable
+    validations:
+      required: true
+  - type: textarea
+    id: relevant-logs
+    attributes:
+      label: Relevant logs and/or screenshots/GIFs
+      description: If applicable, paste any relevant logs
+      render: shell
+      placeholder: Please use code blocks (```) to format console output, logs, and code, as it is very hard to read otherwise
+    validations:
+      required: false
+  - type: textarea
+    id: possible-fixes
+    attributes:
+      label: Possible fixes
+      description: If you can, link to the line of code that might be responsible for the problem
+      render: shell
+    validations:
+      required: false
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running?
+      options:
+        - 1.4.9 (Default)
```

Comment:
- project versin shouldn't be dynamic ? Or simply put a text input instead of selet input...
- multiple : true ?